Services/source/communication/communication_service.c

```diff
@@ -591,7 +592,7 @@ SAT_returnState communication_service_app(csp_packet_t *packet) {
       break;
 
     case UHF_GET_FULL_STAT:
-      status = HAL_UHF_getStatus(&U_stat.status_ctrl) +
+      status = HAL_UHF_getStatus(U_stat.status_ctrl) +
```

Comment:
@ArashYzd FYI this was an error

Reply:
Hmmm...that's weird. How was it working so far then? I actually have changed 2-3 of the HAL functions. will look into this one too and push them soon.